diff --git a/setup.py b/setup - Copy.py
index c6068df..75fb711 100644
--- a/setup.py
+++ b/setup - Copy.py	
@@ -104,7 +104,7 @@ if sys.platform.startswith('win'):
         extra_include_dirs += ['lib\\win32-stdint']
     extra_include_dirs += ['lib\\win32']
     extra_compile_args += [r'/FIPython.h']
-    libraries += ['Advapi32']
+    libraries += ['Advapi32', 'ntdll']
 
 
 # Capture setup.py configuration for later use by cffi, otherwise the
